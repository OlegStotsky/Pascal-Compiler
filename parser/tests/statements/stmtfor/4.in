var
	i : integer;
	startVal : integer;
	endVal : integer;

begin
	for i:=startVal to endVal do
	begin
	end;
end;