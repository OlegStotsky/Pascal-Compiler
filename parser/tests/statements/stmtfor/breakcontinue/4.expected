Error at line 2, column 3 : continue must be called inside the loop
